wiki-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -183,15 +184,21 @@ export default {
         spaceURL: self.spaceURL,
         toolbarLocation: 'top',
         extraAllowedContent: 'img[style,class,src,referrerpolicy,alt,width,height]; span(*)[*]{*}; span[data-atwho-at-query,data-atwho-at-value,contenteditable]; a[*];i[*]',
-        removeButtons: 'Subscript,Superscript,Cut,Copy,Paste,PasteText,PasteFromWord,Undo,Redo,Scayt,Unlink,Anchor,Table,HorizontalRule,SpecialChar,Maximize,Source,Strike,Outdent,Indent,BGColor,About',
+        removeButtons: '',
         toolbar: [
-          { name: 'format', items: ['Format'] },
-          { name: 'basicstyles', items: [ 'Bold', 'Italic', 'Underline', 'Strike', '-', 'RemoveFormat'] },
-          { name: 'paragraph', items: [ 'NumberedList', 'BulletedList', '-', 'Blockquote' ] },
-          { name: 'fontsize', items: ['FontSize'] },
-          { name: 'colors', items: [ 'TextColor' ] },
-          { name: 'align', items: [ 'JustifyLeft', 'JustifyCenter', 'JustifyRight', 'JustifyBlock'] },
-          { name: 'links', items: [ 'simpleLink', 'selectImage', 'Video'] },
+          { name: 'document', items: [ 'Source', '-', 'Save', 'NewPage', 'ExportPdf', 'Preview', 'Print', '-', 'Templates' ] },
```

Comment:
@lbengayed  are you sure you didn't forget to remove the "ExportPdf" declaration here ?

CC @mkrout

Reply:
Yes @plamarque ,I remove the plugin added for exportPDF, it didn't appear in CKEditor toolbar, You can verify in this Lien https://notes-ft.exoplatform.org/portal/dw/notes-editor?parentNoteId=1, 
THis 'ExportPdf' is just a call if the plugin exportPDF added, it will appear in CKEditor toolbar